libs/facilities/backend/timeseries/src/lib/dto/query.dto.ts

```diff
@@ -47,5 +55,6 @@ export class GetTimeSeriesQueryDto implements IGetTimeseriesQuery {
 
 	@IsOptional()
 	@IsBoolean()
+	@Transform(({ value }) => value === 'true')
```

Comment:
Think about implementing a custom validator for this like here the `ToBoolean` would greate easen later implementation.

https://stackoverflow.com/a/66697810

Reply:
I like ur idea, but since i think this query will later be removed i would just leave it like it is right now